testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -306,9 +304,10 @@ impl TxEmitter {
 
     fn make_client(&self, instance: &Instance) -> JsonRpcAsyncClient {
         JsonRpcAsyncClient::new(
-            self.http_client.clone(),
```

Comment:
We actually want to reuse same HttpClient for each JsonRpcAsyncClient, why change it to creating new HttpClient every time?

Reply:
yeah, you are right, fixed.